.travis.yml

```diff
@@ -17,3 +17,5 @@ install:
 
 script:
 - bundle exec jekyll build
+- find ./_site -empty ! -name error.log
+- find ./_site -empty ! -name error.log | read && false
```

Comment:
Like #537, I'm trying to be explicit about return status, but this would likely work as well:

```bash
find ./_site -empty ! -name error.log | ! read
```